src/extensions/step_handlers/script_handler/src/script_handler.cpp

```diff
@@ -572,6 +616,26 @@ static ADUC_Result ScriptHandler_PerformAction(const std::string& action, const
         Log_Debug("##########\n# ADU-SHELL ARGS:\n##########\n %s", ss.str().c_str());
     }
 
+    if (prepareArgsOnly)
+    {
+        for (const auto& a : aduShellArgs)
+        {
+            if (a[0] != '-')
+            {
+                ss << " \"" << a << "\"";
```

Comment:
what if 'a' contains a quote?

Reply:
Added code to quote using ' for the case that the string contains "